README.rst

```diff
@@ -110,7 +110,7 @@ where you have authored all of the code. If you do this on work time
 make sure your employer is cool with this. We also have a `Contributor Guide
 <https://lisa-linux-integrated-system-analysis.readthedocs.io/en/master/contributors_guide.html>`__
 
-.. |Travis status| image:: https://travis-ci.org/ARM-software/lisa.svg?branch=master
-   :target: https://travis-ci.org/ARM-software/lisa
+.. |CI status| image:: https://github.com/ARM-software/lisa/.github/workflows/test.yml/badge.svg?branch=master
```

Comment:
This gives me a 404 Not found

Reply:
The PR is not merged so the file is yet available in the repo